packages/story-editor/src/components/library/panes/shapes/shapePreview.js

```diff
@@ -138,6 +139,8 @@ function ShapePreview({ mask, isPreview, index }) {
       mask: {
         type: mask.type,
       },
+      // Rectangles have unlocked aspect ratio by default.
+      lockAspectRatio: MaskTypes.RECTANGLE !== mask.type,
```

Comment:
Specific UX request. I suppose it's because some other shapes might start looking odd when not locked but the rectangle will likely be changed anyway to a non-square. Just assuming here of course.